### PR TITLE
Input

### LINKED ISSUE #18: Update: Input.html
adding sidebar contents in Input.html

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. label Input
LabelInput
2. Input with Error styling
ErrorInput
3.text-box
TextBox

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
all look good -

File: docs/Inputs/Input.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        <div class="docs-wrapper">
+            <header>
+                <div class="container">
+                    <div class="logo">
+                        <img src="../../assets/brand-logo.svg" alt="Logo">
+                        <h3>Pegasus</h3>
+                    </div>
+                    <nav class="links">
+                        <ul>
+                           <li><a href="https://github.com/RahulSingh9131">Github</a></li>
```
Comment (shubhamkumar648) [inline docs/Inputs/Input.html:24]: add target _blanck attributes

**Review by ttahm3d — APPROVED** [review]
Looks good to merge.

**Review by shubhamkumar648 — APPROVED** [review]
Remaining looks good

File: docs/Inputs/Input.html (hunk 1 above)
Comment (RahulSingh9131) [reply docs/Inputs/Input.html:24]: yes i will add that after every component is done

**Pull request merged**